.editorconfig

```diff
@@ -161,9 +170,13 @@ dotnet_naming_symbols.any_async_methods.required_modifiers = async
 
 # Rules
 
-dotnet_naming_rule.constant_fields_should_be_pascal_case.severity = error
-dotnet_naming_rule.constant_fields_should_be_pascal_case.symbols  = constant_fields
+dotnet_naming_rule.constant_fields_should_be_pascal_case.symbols = constant_fields
 dotnet_naming_rule.constant_fields_should_be_pascal_case.style = pascal_case_style
+dotnet_naming_rule.constant_fields_should_be_pascal_case.severity = error
+
+dotnet_naming_rule.local_constant_should_be_uppercase.symbols = local_constant
+dotnet_naming_rule.local_constant_should_be_uppercase.style = uppercase_with_underscore_separator
```

Comment:
All screaming caps and underscores? Not C# style, and certainly not something we want to enforce in a OSS project, where this will make dotnet developers squirm.

I think should be PascalCase in const class members (public or private). For consts declared in methods like these, it really doesn't matter whether it's pascal or camel case, but if it has to be enforced at all, it should be consistent (PascalCase as @stephentoub says).

Reply:
with PascalCase isn't it harder to distinguish constants from Properties?

Reply:
changed to PascalCase. I find the convention quite confusing, plus it's one more reason to enforce `this.`